packages/gafl-webapp-service/src/server.js

```diff
@@ -179,6 +180,12 @@ const init = async () => {
    */
   server.decorate('request', 'cache', cacheDecorator(sessionCookieName))
 
+  server.decorate('toolkit', 'redirectWithLanguageCode', function (request, redirect) {
+    const uriWithLanguage = addLanguageCodeToUri(request, redirect)
```

Comment:
There are still a few instances where we use `addLanguageCodeToUri` without a redirect, for example when we are determining where the "Back" link in the journey should go. Otherwise I would have just rolled it in here.